genome_browser/_genome_diagram.py

```diff
@@ -88,25 +90,24 @@ def draw(self):
 
                 # Figure annotations will be applied to the last ax in offset
                 # coordinates in a lightgray text. Clipping is ignored as the
-                # text clearly cips with the axes outboard frame.
-                if self.annotation is not None:
-                    ax.annotate(
-                        xy=(1, 0),
-                        xycoords='axes fraction',
-                        s=self.annotation,
-                        xytext=(0, -60),
-                        textcoords='offset points',
-                        va='bottom',
-                        ha='right',
-                        color='0.6',
-                        clip_on=False)
+                # text clearly clips with the axes outboard frame.
+                ax.annotate(
+                    xy=(1, 0),
+                    xycoords='axes fraction',
+                    text=self.annotation,
+                    xytext=(0, -22.5),
+                    textcoords='offset points',
+                    va='top',
+                    ha='right',
+                    color='0.6',
+                    clip_on=False)
 
             # Provide simple logic for plotting a track annotation in the
             # top left of each track. The position will remain constant as its
             # defined using proportional values of the ax's x and y limits.
-            if track.annotate is True and track.name is not None:
+            if track.annotate and track.name:
                 ax.annotate(
-                    s=track.name,
+                    text=track.name,
```

Comment:
![Screenshot from 2022-05-20 15-49-26](https://user-images.githubusercontent.com/13955209/169542166-48153158-1dc0-40ba-ab8b-c3d59cfc7ebc.png)

`matplotlib` API update, so I've gone along with it.